src/client/components/pages/adminLogs.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (C) 2023 Shivam Awasthi
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import AdminLogsTable from './parts/admin-logs-table';
+import PagerElement from './parts/pager';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+
+class AdminLogsPage extends React.Component {
+	constructor(props) {
```

Comment:
How would you feel about making this component a functional component instead of a class?
It seems very simple to convert and we might as well try to stay up to date with the newer syntax when we can.
Let me know if you need a hand.

Reply:
fixed!